<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ytxTester</class>
 <widget class="QMainWindow" name="ytxTester">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>254</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ytxTester</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="message"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="radioInt">
          <property name="text">
           <string>DEC</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioHex">
          <property name="text">
           <string>HEX</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="sendSerial">
        <property name="text">
         <string>Send SysEx</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPortOut">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>254</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="optionsMenu">
    <property name="title">
     <string>Options</string>
    </property>
   </widget>
   <addaction name="optionsMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionInput_port">
   <property name="text">
    <string>Input port</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
